cmake/modules/FindFFMPEG.cmake

```diff
@@ -109,12 +109,12 @@ macro(buildFFMPEG)
 "#!${BASH_COMMAND}
 if [[ $@ == *${APP_NAME_LC}.bin* || $@ == *${APP_NAME_LC}${APP_BINARY_SUFFIX}* || $@ == *${APP_NAME_LC}.so* || $@ == *${APP_NAME_LC}-test* ]]
 then
-  avformat=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libavcodec`
-  avcodec=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libavformat`
-  avfilter=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libavfilter`
-  avutil=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libavutil`
-  swscale=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libswscale`
-  swresample=`PKG_CONFIG_PATH=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libswresample`
+  avformat=`PKG_CONFIG_PATH=${DEPENDS_PATH}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static libavcodec`
```

Comment:
I know it was wrong before, but this should be `avformat`
and `avcodec` below

Reply:
Are you sure? testing on osx i get this 

```
MacBook-Pro-2 ~ % PKG_CONFIG_PATH=/Users/Shared/xbmc-depends/macosx12.3_x86_64-target-debug/lib/pkgconfig /Users/Shared/xbmc-depends/arm-darwin21.4.0-native/bin/pkg-config --libs --static avformat
Package avformat was not found in the pkg-config search path.
Perhaps you should add the directory containing `avformat.pc'
to the PKG_CONFIG_PATH environment variable
No package 'avformat' found
MacBook-Pro-2 ~ % PKG_CONFIG_PATH=/Users/Shared/xbmc-depends/macosx12.3_x86_64-target-debug/lib/pkgconfig /Users/Shared/xbmc-depends/arm-darwin21.4.0-native/bin/pkg-config --libs --static libavformat
-L/Users/Shared/xbmc-depends/macosx12.3_x86_64-target-debug/lib -lavformat -lm -lbz2 -lz -lgnutls -lgmp -ldl -lhogweed -lgmp -lnettle -lavcodec -lm -llzma -ldav1d -lz -framework AudioToolbox -pthread -framework VideoToolbox -framework CoreFoundation -framework CoreMedia -framework CoreVideo -framework CoreServices -lswresample -lm -lavutil -pthread -lm -framework VideoToolbox -framework CoreFoundation -framework CoreMedia -framework CoreVideo -framework CoreServices
brent@Brents-MacBook-Pro-2 ~ % 
```

Reply:
sorry, ofc it's libavformat.
point was:
 **avformat**`=`PKG_CONFIG_PATH=${DEPENDS_PATH}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static **libavformat**
vs
avformat=PKG_CONFIG_PATH=${DEPENDS_PATH}/lib/pkgconfig ${PKG_CONFIG_EXECUTABLE} --libs --static _libavcodec_